Doc/library/functions.rst

```diff
@@ -1713,8 +1713,8 @@ are always available.  They are listed here in alphabetical order.
 
 
 .. _func-range:
-.. class:: range(stop)
-           range(start, stop, step=1)
+.. class:: range(stop, /)
+           range(start, stop, step=1, /)
```

Comment:
I think enumerating all three cases is slightly nicer. The `step` argument is for advanced use-cases. Thoguhts @terryjreedy?
```suggestion
           range(start, stop, /)
           range(start, stop, step, /)
```

Reply:
I generally prefer fewer lines, partly from experience with IDLE call tips (which would be similar in other IDEs).  Serhiy generally went with fewer in his PR, as he did here, except where really needed.  Two are really needed here, since the true Python-implementaton signature `(start_or_stop, stop_if_needed=None, step=1, /)` is definitely too baroque for the doc.

Given your approval, I will merge this and backport (as Serhiy plans to backport his changes).